packages/compass-import-export/package.json

```diff
@@ -1,171 +1,78 @@
 {
   "name": "@mongodb-js/compass-import-export",
-  "productName": "Compass Import/Export",
-  "version": "6.21.0",
-  "apiVersion": "3.0.0",
-  "description": "Compass Import/Export Plugin",
-  "homepage": "https://github.com/mongodb-js/compass",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mongodb-js/compass.git"
+  "description": "Compass Import/Export",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
   },
+  "private": true,
   "bugs": {
     "url": "https://jira.mongodb.org/projects/COMPASS/issues",
     "email": "compass@mongodb.com"
   },
-  "main": "lib/index.js",
+  "homepage": "https://github.com/mongodb-js/compass",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/compass.git"
+  },
+  "files": [
+    "dist"
+  ],
+  "license": "SSPL",
+  "main": "dist/index.js",
   "exports": {
-    "webpack": "./src/index.js",
-    "require": "./lib/index.js"
+    "webpack": "./src/index.ts",
+    "require": "./dist/index.js",
+    "import": "./dist/.esm-wrapper.mjs"
   },
+  "types": "./dist/index.d.ts",
   "scripts": {
-    "clean": "rimraf lib",
-    "precompile": "npm run clean",
-    "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js",
-    "start": "webpack-dev-server --config ./config/webpack.dev.config.js",
-    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
-    "pretest": "mongodb-runner start --port=27018",
-    "posttest": "mongodb-runner stop --port=27018",
-    "test:watch": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" --watch",
-    "test:dev": "cross-env NODE_ENV=test mocha-webpack",
-    "cover": "nyc npm run test",
-    "check": "npm run lint && npm run depcheck",
+    "bootstrap": "npm run compile",
     "prepublishOnly": "npm run compile",
-    "lint": "eslint \"./src/**/*.{js,jsx}\" \"./test/**/*.js\"",
+    "compile": "tsc -p tsconfig.json && gen-esm-wrapper . ./dist/.esm-wrapper.mjs",
```

Comment:
If we want to drop webpack build, then we need to remove the config files too (but this will also not work when trying to bundle less files), if we want to keep it and we are updating the plugin to new configuration, then we should use new shared webpack config (you can use create-workspace output for the plugin workspace as a reference for what configs and npm scripts should be part of the package)

Reply:
I basically re-created the `compass-import-export` package with `create-workspace.js` script and then copied the files over and fixed eslint errors. Basically this PR will prepare the repo to use Typescript for the Exporter refactory

Reply:
When you were generating it, did you marked this package as plugin? Seems like some default plugin configs like the one for webpack are missing. Or are we intentionally not generating this package as plugin?

Reply:
I think I chose "no", I'll try regenerating it as plugin